blazor/appearance/icons.md

```diff
@@ -11,6 +11,11 @@ documentation: ug
 
 The Syncfusion Blazor library provides the set of `base64` formatted font icons which are being used in the Syncfusion Blazor components. These icons can be utilized in the web applications using `SfIcon` component or `e-icons` class.
 
+To get start quickly with Blazor Icon component, you can check on this video:
```

Comment:
To get start quickly with the Blazor Icon component, you can check on this video: